### ISSUE TITLE
XPath NoSuchElementException on get_users_through_dialog

### ISSUE THREAD

**Issue by cyberpz** [InstaPy/InstaPy#5246]
```
Traceback (most recent call last):
  File "bot.py", line 38, in <module>
    session.follow_user_followers([lista], amount=amount, randomize=True)
  File "/Users/Apple/Library/Python/2.7/lib/python/site-packages/instapy/instapy.py", line 3240, in follow_user_followers
    self.logfolder)
  File "/Users/Apple/Library/Python/2.7/lib/python/site-packages/instapy/unfollow_util.py", line 1084, in get_given_user_followers
    logger, logfolder)
  File "/Users/Apple/Library/Python/2.7/lib/python/site-packages/instapy/unfollow_util.py", line 807, in get_users_through_dialog
    dialog = browser.find_element_by_xpath(dialog_address)
  File "/Users/Apple/Library/Python/2.7/lib/python/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/Users/Apple/Library/Python/2.7/lib/python/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/Users/Apple/Library/Python/2.7/lib/python/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/Users/Apple/Library/Python/2.7/lib/python/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//body/div[3]/div/div[2]"}
  (Session info: chrome=78.0.3904.108)
```

I will update you as I try to fix it

**Comment by Peppuz** [comment]
So was trying to replace the `xpath["get_users_through_dialog"]["find_dialog_box"]` with the new xpath`//body/div[4]/div`

**Comment by cyberpz** [comment]
Sorry for closing, pressed the wrong button.
So was trying to replace the xpath["get_users_through_dialog"]["find_dialog_box"] with the new xpath `//body/div[4]/div` but it does't collect users, so the bot won't go further on following user collected. It seems to not crash tho.


### What is causing this? 
```
 session = InstaPy(username="x",password="x")
        with smart_run(session):
                session.follow_user_followers(listA, amount=amount, randomize=True)
```

PS: @analyticsdept good job on the xpath isolation, simple and clean.

**Comment by cyberpz** [comment]
Fixed on PR #5252
